src/Microsoft.AspNetCore.Razor.Language/DefaultRazorTagHelperBinderPhase.cs

```diff
@@ -53,7 +53,10 @@ protected override void ExecuteCore(RazorCodeDocument codeDocument)
             }
 
             var errorSink = new ErrorSink();
-            var rewriter = new TagHelperParseTreeRewriter(tagHelperPrefix, descriptors);
+            var rewriter = new TagHelperParseTreeRewriter(tagHelperPrefix, descriptors)
```

Comment:
Should just have this guy take in the feature flags.

Reply:
Whoy not utilize the ctor?

Reply:
No reason now. Did that when I wasn't passing in featureFlags directly